src/components/TimeSeriesPanel/TimeSeriesChartHeader.tsx

```diff
@@ -39,6 +39,8 @@ import FitScreenIcon from "@mui/icons-material/FitScreen";
 import IsoIcon from "@mui/icons-material/Iso";
 import ScatterPlotIcon from "@mui/icons-material/ScatterPlot";
 import ShowChartIcon from "@mui/icons-material/ShowChart";
+import CameraAltIcon from "@mui/icons-material/CameraAlt";
+import { useDispatch } from "react-redux";
```

Comment:
No dispatch + no redux here please!

Redux code is only in `src/connected`. Here we are in `src/components` which is redux-free to stay reusable for later projects.